apps/modia-soknadsstatus-api/src/main/kotlin/no/nav/modia/soknadsstatus/service/HendelseService.kt

```diff
@@ -60,6 +61,11 @@ class HendelseServiceImpl(
                         HendelseEierDAO(ident = ident, hendelseId = requireNotNull(hendelse?.id)),
                     )
                 }
+            } else {
+                TjenestekallLogg.error(
+                    header = "Klarte ikke å lagre behandling i databasen",
+                    fields = mapOf("behandlingsId" to innkommendeHendelse.behandlingsId, "hendelsesId" to innkommendeHendelse.hendelsesId),
+                )
             }
```

Comment:
Vi burde kaste en feil her i stedet for å logge. Da blir meldingen sendt til DLQ.